roles/static_inventory/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Either to checkpoint the dynamic inventory into a static one
+refresh_inventory: True
+inventory: static
+inventory_dir: ~/openstack-inventory
```

Comment:
Okay, so this now does re-use the existing inventory, but because `inventory_dir` is a special Ansible variable, it overrides the custom vars which means that it's no matter what we set `inventory_dir` to, it will always point to the current inventory.

So I think we should change this back to `inventory_path` (sorry! :-( ) and then in #550 remove (or change the default of) `openstack_inventory_dir`.